share/doc/wake/quickref.md

```diff
@@ -249,17 +249,20 @@ Runners are created using `makeRunner`. `makeRunner` is defined as follows:
 
 Environment packages can be added to a workspace to provide tools (using runners) for running the workflow. They are composed of Wake files that define and publish runners. Environment packages should be defined for each unique running environment to decouple Wake rules from how tools are installed in each environment. You can check out [environment-example-sifive](https://github.com/sifive/environment-example-sifive) as an example of an environment package that contains runners for Wake-based workflows. 
 
-#### sources and Path objects
+#### Sources and `Path` objects
 
-Sources are the set of files in git.
+Sources are the set of files in a Git repository. A `Path` in Wake is either a) a file in Git or b) a file produced from a build step. The `Path` type has no public constructor to help enforce this. To get a `Path` value, use the
 
-A Path is either a) a file in git or b) a file produced from a build step
+* `source: (file: String) => Result Path Error`
+* `sources: (dir: String) => (filterRegexp: RegExp) => Result (List Path) Error`
 
-To get a path, use the source function. For example, all the header files in this directory:
+functions. For example, to get all the header files in the current repository:
 
+```wake
+sources here `.*\.h`
```

Comment:
nit: `here` is deprecated (replaced with `@here`) so we should use that in the docs

Reply:
Thanks for the heads up! [Fixed here.](https://github.com/sifive/wake/pull/1009/commits/4128f7caa46084157a37e748456da2c13b9612f1)

Reply:
[I went ahead and also updated all `here` instances with `@here` in the Wake tutorial.](https://github.com/sifive/wake/pull/1009/commits/d292499c08065921b0ed8a2d716d9ee81a14a51f)